dynawo/sources/Modeler/Common/test/Test.cpp

```diff
@@ -688,8 +688,9 @@ TEST(ModelerCommonTest, VariableAlias) {
   ASSERT_EQ(variableAlias3->getNegated(), true);
   ASSERT_EQ(variableAlias3->getIndex(), varIndex);
 
+#ifndef _MSC_VER
   EXPECT_ASSERT_DYNAWO(VariableAliasFactory::create(aliasName3, dynamic_pointer_cast<VariableNative> (variableAlias2), DISCRETE));
```

Comment:
Why is this line not working on Windows?

Reply:
it uses ::testing::KilledBySignal(SIGABRT) that is not available on Windows